shared/actions/chat/manage-thread.js

```diff
@@ -42,7 +42,7 @@ function* _startConversation(action: ChatGen.StartConversationPayload): Saga.Sag
   if (forceImmediate && existing) {
     const newID = yield Saga.call(Shared.startNewConversation, existing.get('conversationIDKey'))
     yield Saga.put(ChatGen.createSelectConversation({conversationIDKey: newID}))
-  } else if (existing && !temporary) {
+  } else if (existing) {
```

Comment:
If we don't have pending conversations anymore, is the else case below still needed? Also maybe can get rid of `createAddPending` and the `temporary` param in the payload